src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+public abstract class Figure implements Drawable, AreaCalculator {
+    protected String color;
```

Comment:
in this class should be only color, other fields should be in Figure heirs where it's appropriate